docs/testing/releases/380.md

```diff
@@ -0,0 +1,21 @@
+## Testing notes and ZIP for release 3.8.0
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-gutenberg-products-block/files/5516903/woocommerce-gutenberg-products-block.zip)
+
+## The following changes affect the feature plugin only.
+
+### Show the phone number field in the billing section when shipping is disabled in settings. ([3376](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3376))
+
+Moves the rendering of address fields to a single file so that PhoneNumber (new component) can be rendered in either billing or shipping, depending on visibility, without duplication.
+
+0. Ensure the phone number field is enabled in block settings, then test 2 cases:
```

Comment:
I would specify this is referring to the Checkout block, to avoid confusion:
```suggestion
0. Ensure the phone number field is enabled in Checkout block settings, then test 2 cases:
```